frontend/src/pages/notifications/mock-data.js

```diff
@@ -1,42 +1,42 @@
 export const sampleUser = {
```

Comment:
should we remove this?

Reply:
Yep you are right I have removed all of the mock data